src/Indice.Features.Messages.AspNetCore/Endpoints/DistributionListsApi.cs

```diff
@@ -82,6 +85,35 @@ public static RouteGroupBuilder MapDistributionLists(this IEndpointRouteBuilder
              .WithSummary("Removes an existing contact from the specified distribution list.")
              .WithDescription(DistributionListsHandlers.REMOVE_CONTACT_FROM_DISTRIBUTION_LIST_DESCRIPTION);
 
+        group.MapGet("{distributionListId}/export", DistributionListsHandlers.BulkExportContactsFromDistributionList)
+            .WithName(nameof(DistributionListsHandlers.BulkExportContactsFromDistributionList))
+            .WithSummary("Bulk exports contacts from a specified distribution list in CSV file.")
+            .WithDescription(DistributionListsHandlers.BULK_EXPORT_CONTACTS_FROM_DISTRIBUTION_LIST)
+            .WithOpenApi(x =>
```

Comment:
This is not the usual way we do this. Take a look in media or attachments. This is not needed